beta-src/src/utils/voteStateArray.ts

```diff
@@ -0,0 +1,28 @@
+import VoteType from "../types/Vote";
+
+interface SingleVote {
+  vote: string;
+  status: boolean;
+}
+
+type statusArray = SingleVote[];
+
+export default function voteStateArray(voteState: VoteType): statusArray {
+  const voteArray: statusArray = [];
+
+  const keys = Object.keys(voteState);
+  const values = Object.values(voteState);
+
+  for (let i = 0; i <= keys.length; i += 1) {
+    if (Object.prototype.hasOwnProperty.call(values, i)) {
+      const singleVote: SingleVote = {
+        vote: keys[i],
+        status: values[i],
+      };
+
+      voteArray.push(singleVote);
+    }
+  }
+
+  return voteArray;
+}
```

Comment:
I don't understand why this is needed? Aren't you able to map the `voteState` object directly in `WDCommandButtons.tsx`?

Reply:
File deleted.. use Object.entries() instead